### DISCUSSION TITLE
IGNORE: EXISTING FEATURE "Update All Scripts" button

### DISCUSSION THREAD

**Discussion by Starcalled** [Tampermonkey/tampermonkey#2203] [Ideas]
SUGGESTION: Add a button to the Settings or Utilities tab that will sequentially check each installed script for updates.

REASONING: It's mildly inconvenient having to go to each individual script, click on its settings menu, click update, and wait for it to check, close the tab, and then repeat that task 20x for every single one of your scripts. It seems like an obvious feature to me, I can't really see a downside to adding it. 

If John Tampermonkey ends up seeing this, please consider adding it ❤️ that's all I guess, thank you for reading

**Comment by derjanb** [comment]
The utilities section at the popup menu includes an item "Check userscripts for updates".
You can also select all scripts (Ctrl+a) and trigger an update at the dashboard.

|||
|--|--|
|![image](https://github.com/user-attachments/assets/f9e42a46-e98e-438e-80d3-66ab82b5e085)|![image](https://github.com/user-attachments/assets/63fad418-fea7-4c37-8812-cbf10d7a0f5a)|

**Reply by Starcalled** [reply]
Damb, guess I'm just blind. Thank you for the response !!
